core/services/ardupilot_manager/mavlink_comm/MavlinkComm.py

```diff
@@ -0,0 +1,80 @@
+import json
+import os
+import time
+from typing import Any, Dict, Optional
+
+import requests
+
+from exceptions import (
+    FetchUpdatedMessageFail,
+    MavlinkMessageReceiveFail,
+    MavlinkMessageSendFail,
+)
+
+
+class MavlinkMessenger:
+    def __init__(self) -> None:
+        self.system_id = int(os.environ.get("MAV_SYSTEM_ID", 1))
+        self.component_id = int(os.environ.get("MAV_COMPONENT_ID_ONBOARD_COMPUTER4", 194))
```

Comment:
were you going to set this environment variables somewhere?

Reply:
This is set [here](https://github.com/bluerobotics/companion-docker/blob/a022660a8069dde984dc81f71c0fbcc3a4e24e68/core/start-companion-core#L8)